docs/api/vega-wallet/reference/core/json-rpc.md

```diff
@@ -2190,7 +2185,7 @@ undefined
     "value": {
         "activeConnections": [
             {
-                "hostname": "console.vega.xyz",
+                "hostname": "site.web.com",
```

Comment:
```suggestion
                "hostname": "example.com",
```

**nitpick (non-blocking)**: `example.com` is reserved for exactly this, web.com is.. something AIy.